internal/position.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 type Position struct {
-	ID     uuid.UUID       `json:"id"`
+	ID     uuid.UUID       `json:"ID"`
```

Comment:
Why do you use uppercase for that particular filed?

Reply:
I didn't want it, but it was misclick